src/ivalue_manager.rs

```diff
@@ -28,10 +28,26 @@ use crate::array_index::ArrayIndex;
 use bson::decode_document;
 use std::io::Cursor;
 
+///
+/// Either hold reference to the IValue stored in a RedisJSON key
+/// or a value that was parsed from a String
+///
+enum IValueHolder<'a> {
+    Ref(&'a mut RedisJSON<IValue>),
+    Value(IValue),
```

Comment:
We are still running tests also with SERDE_JSON manager (with module arg `JSON_BACKEND SERDE_JSON`)
(e.g., [test](https://app.circleci.com/pipelines/github/RedisJSON/RedisJSON/5141/workflows/0c89a616-ff98-4f83-9d0b-fd9590af2c0e/jobs/19318?invite=true#step-114-571))

Should we drop `ManagerType::SerdeValue` or should we handle it as `Value` in addition to `IValue`?
(add it to the Manager?)

Reply:
I think we can drop serde_json for the next minor version

Reply:
@MeirShpilraien do you see a reason to keep it?